### PR TITLE
#52 Azure cache invalidate via pooling

### PR DESCRIPTION
**Pull request by ksavosteev**
Blob monitoring mostly taken from the old storefront. 
Added a custom implementation of IChangeToken - BlobChangeToken.
For each blob file BlobChangeToken is created and placed in ConcurrentDictionary (MonitorFileSystemChanges method).
Each iteration of monitor updates (if is already created) blob token's LastModified date so each token can have its own changed state.
After token obtains 'changed' state it's discarded and a new one is created (Watch method).

### CODE REVIEW

**Comment by asvishnyakov** [comment]
@tatarincev Make review, please

**Comment by tatarincev** [comment]
We have discussed and decided to make another implementation, but @ksavosteev is busy at the moment, and there is no one who could do it

File: VirtoCommerce.Storefront/appsettings.json [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
+            "RethrowLiquidRenderErrors": false
+        },
+        "RequireHttps": {
+            "Enabled": false,
+            "StatusCode": "308",
+            "Port": "443"
+        },
+        "AzureBlobStorage": {
+            "PollForChanges": true,
+            "PollingChangesInterval": "0:0:05"
```
Comment (tatarincev) [inline VirtoCommerce.Storefront/appsettings.json]: Сould you rename this setting to `ChangesPoolingInterval` to be similar in the  `Virtocommerce:ChangesPoolingInterval` setting?

**Comment by tatarincev** [comment]
Released https://github.com/VirtoCommerce/vc-storefront-core/releases/edit/v3.0.7

**Pull request merged**